<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditWidget</class>
 <widget class="QWidget" name="EditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="btnCancel">
       <property name="text">
        <string>Отменить</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-cancel">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnOk">
       <property name="text">
        <string>Принять</string>
       </property>
       <property name="icon">
        <iconset theme="dialog-ok">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="btnZoomOut">
       <property name="text">
        <string>Уменьшить</string>
       </property>
       <property name="icon">
        <iconset theme="zoom-out">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnZoomIn">
       <property name="text">
        <string>Увеличить</string>
       </property>
       <property name="icon">
        <iconset theme="zoom-in">
         <normaloff>.</normaloff>.</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Unicode</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Ширина</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="edtHeight"/>
     </item>
     <item row="1" column="5">
      <widget class="QSpinBox" name="edtDX">
       <property name="minimum">
        <number>-1024</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Высота</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edtUnicode"/>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edtWidth"/>
     </item>
     <item row="1" column="6">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>∆Y</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>∆X</string>
       </property>
      </widget>
     </item>
     <item row="1" column="7">
      <widget class="QSpinBox" name="edtDY">
       <property name="minimum">
        <number>-1024</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Смещение X</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="edtXAdvance"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
